src/validations/request.validation.js

```diff
@@ -0,0 +1,49 @@
+import ResponseService from '../services/response.service';
+import JwtService from '../services/jwt.service';
+import UserService from '../services/user.service';
+import TripService from '../services/trip.service';
+
+/**
+ * @param {req} req
+ * @param {req} res
+ * @param {next} next
+ * @returns {validation} this function validate request route
+*/
+async function requestValidation(req, res, next) {
+  const signInUser = JwtService.verifyToken(req.headers.authorization);
+  const userId = req.params.user_id;
+
+  const regex = /[^0-9]/g;
```

Comment:
Can you validate with hapi/joi? I think it would reduce the lines of code in this file.